pkg/apk/implementation.go

```diff
@@ -549,18 +549,26 @@ func (a *APK) FixateWorld(ctx context.Context, sourceDateEpoch *time.Time) error
 			return fmt.Errorf("cannot install due to conflict with %s", pkg)
 		}
 	}
+	allInstPkgs := make([]InstallablePackage, len(allpkgs))
+	for i, pkg := range allpkgs {
+		allInstPkgs[i] = pkg
+	}
```

Comment:
Is this just to appease go's type system? Like `[]*RepositoryPackage` is not a `[]InstallablePackage`?

If so, could you add a comment to that effect?